package.json

```diff
@@ -56,13 +56,13 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "prop-types": "^15.5.10",
-    "react": "^15.6.1",
-    "react-dom": "^15.6.1"
+    "react": "^16.0.0",
```

Comment:
Why: https://til.hashrocket.com/posts/oqjntperif-render-an-array-of-elements-with-react-16